scipy/fft/__init__.py

```diff
@@ -74,7 +75,8 @@
     'hfft', 'ihfft', 'hfft2', 'ihfft2', 'hfftn', 'ihfftn',
     'fftfreq', 'rfftfreq', 'fftshift', 'ifftshift',
     'next_fast_len',
-    'dct', 'idct', 'dst', 'idst', 'dctn', 'idctn', 'dstn', 'idstn']
+    'dct', 'idct', 'dst', 'idst', 'dctn', 'idctn', 'dstn', 'idstn',
+    'set_backend', 'set_global_backend', 'register_backend']
```

Comment:
These should be added to the docstring above, probably in their own subheading like:
```
Backend control
===========
```
Or whatever else you think is suitable.

I am surprised that the `refguide` did not complain about this omission. Probably because it's missing here:

https://github.com/scipy/scipy/blob/master/tools/refguide_check.py#L73

It's probably worth going to the `scipy` root and doing a `git grep fftpack`, it might show other places that there is some module list that should have `scipy.fft` added to it.

Reply:
autosummary added. fft added to refguide and `git grep` also found `_build_util/tests/test_circular_imports.py`.